pinot-common/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -109,3 +109,14 @@ SqlNode SqlPhysicalExplain() :
             nDynamicParams);
     }
 }
+SqlNode SqlShowTables() :
+{
+    SqlParserPos pos;
+}
+{
+    <SHOW> { pos = getPos(); }
```

Comment:
Do we need to modify the parser? Is there a standard SQL way of showing tables?

Reply:
I don't think `SHOW` keyword is part of the parser, so I had to add it.